### ISSUE TITLE
Add setup instructions for first VM boot

### ISSUE THREAD

**Issue by daPhipz** [quickgui/quickgui#31]
There are missing instructions on how to install macOS VMs with Quickgui. Yes, there is https://github.com/wimpysworld/quickemu#macos-guest - but I can imagine that this is the last place where a "GUI-first" user will look for instructions.

I also think that adding SPICE setup instructions (See [Ubuntu Guest](https://github.com/wimpysworld/quickemu#ubuntu-guest)) would be nice to show at first boot of the VM.

**Comment by ymauray** [comment]
Good call. Maybe a "help" screen in the app, on top of info in the README?

**Comment by PixsaOJ** [comment]
This issue is outdated, none of the link references are valid in OP's description
